src/main/java/com/thealgorithms/strings/Pangram.java

```diff
@@ -15,6 +17,22 @@ public static void main(String[] args) {
         assert !isPangram("\u0000/\\");
     }
 
+    /**
+     * Checks if a String is considered a Pangram
+     *
+     * @param s The String to check
+     * @return {@code true} if s is a Pangram, otherwise {@code false}
+     */
+    public static boolean isPangramOrNot(String s) {
+        // an approach using Java Collection Framework
```

Comment:
```suggestion
     * @return {@code true} if s is a Pangram, otherwise {@code false}
     */
     // alternative approach using Java Collection Framework
    public static boolean isPangramOrNot(String s) {
```

Reply:
I have committed the above stated changes